pkgs/by-name/me/mediasynclite/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gtk3
+, glib
+, gsettings-desktop-schemas
+, pkg-config
+, curl
+, openssl
+, jansson
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mediasynclite";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "iBroadcastMediaServices";
+    repo = "MediaSyncLiteLinux";
+    rev = "d0d5caa2a1bb260f8081e4cf49cb0219b123f682";
+    hash = "sha256-mS2e6zZutSEAr5gCS58t0Nzg8dCJgn/yoFMmbgnIM2I=";
+    };
+
+  buildInputs = [
+   gtk3
+   glib
+   gsettings-desktop-schemas
+   pkg-config
+   curl
+   openssl.dev
+   jansson
+   wrapGAppsHook
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  prePatch = ''
+    substituteAllInPlace ./src/ibmsl.c
```

Comment:
What is replaced here?

Reply:
> What is replaced here?

The @out@ [here](https://github.com/tobz619/MediaSyncLiteLinuxNix/blob/d0d5caa2a1bb260f8081e4cf49cb0219b123f682/src/ibmsl.c#L671)

Reply:
But in such case you need `--subst-var out`